configs/sites/tier1/aws-pcluster/README.md

```diff
@@ -217,51 +217,7 @@ sudo usermod -aG docker $USER
 exit
 ```
 
-4. Build ecflow outside of spack to be able to link against OS boost
-```
-mkdir -p /home/ubuntu/jedi/ecflow-5.8.4/src
-cd /home/ubuntu/jedi/ecflow-5.8.4/src
-wget https://github.com/ecmwf/ecflow/archive/refs/tags/5.8.4.tar.gz
-tar -xvzf 5.8.4.tar.gz
-export WK=/home/ubuntu/jedi/ecflow-5.8.4/src/ecflow-5.8.4
-export BOOST_ROOT=/usr
-
-# Build ecFlow
-cd $WK
-mkdir build
-cd build
-cmake .. -DPython3_EXECUTABLE=/usr/bin/python3 -DENABLE_STATIC_BOOST_LIBS=OFF -DCMAKE_INSTALL_PREFIX=/home/ubuntu/jedi/ecflow-5.8.4 2>&1 | tee log.cmake
-make -j10 2>&1 | tee log.make
-make install 2>&1 | tee log.install
-
-# Create a modulefiles directory and the "ecflow/5.8.4" module file.
-mkdir -p /home/ubuntu/jedi/modulefiles/ecflow
-
-cat << 'EOF' > /home/ubuntu/jedi/modulefiles/ecflow/5.8.4
-#%Module1.0
-
-module-whatis "Provides an ecflow-5.8.4 server+ui installation for use with spack."
-
-conflict ecflow
-
-proc ModulesHelp { } {
-puts stderr "Provides an ecflow-5.8.4 server+ui installation for use with spack."
-}
-
-# Set this value
-set ECFLOW_PATH "/home/ubuntu/jedi/ecflow-5.8.4"
-
-prepend-path PATH "${ECFLOW_PATH}/bin"
-prepend-path LD_LIBRARY_PATH "${ECFLOW_PATH}/lib"
-prepend-path LIBRARY_PATH "${ECFLOW_PATH}/lib"
-prepend-path CPATH "${ECFLOW_PATH}/include"
-prepend-path CMAKE_PREFIX_PATH "${ECFLOW_PATH}"
-prepend-path PYTHONPATH "${ECFLOW_PATH}/lib/python3.10/site-packages"
-EOF
-
-```
-
-5. Install lmod. This step must be done as `root`.
```

Comment:
This is a weird diff, the change is presented on line 220

Reply:
all good, the itemization changed from 5 to 4, therefore it looks like a diff